### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by raphox**
Insert reference to pending template

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -386,7 +386,8 @@ function make_templates(dataset) {
 		header: make(dataset.header),
 		footer: make(dataset.footer),
 		suggestion: make(dataset.suggestion || dataset.template),
-		notFound: make(dataset.notFound || dataset.empty)
+		notFound: make(dataset.notFound || dataset.empty),
+		pending: make(dataset.pending || dataset.pending)
```
Comment (sergeyt) [inline index.js]: please consider to fix duplicated `dataset.pending`

**Comment by raphox** [comment]
Hi @sergeyt,

I can send you a new PR (https://help.github.com/articles/merging-a-pull-request/) or do you prefer me to do a local rebase?

Regards.

**Comment by sergeyt** [comment]
hi @raphox, new PR is ok.

**Comment by raphox** [comment]
@sergeyt to can use this template I need set data to inputs by:

```javascript
template.$('.typeahead').data('options', { async: true });
```
because this check https://github.com/sergeyt/meteor-typeahead/blob/master/typeahead.bundle.js#L1587

Regards.

**Comment by raphox** [comment]
@sergeyt maybe you can set async data in https://github.com/sergeyt/meteor-typeahead/blob/master/index.js#L184 conditional

**Comment by sergeyt** [comment]
@raphox what about explicit use of `data-async` attribute? also when `pending` template is used `async` option is true by default.

**Comment by sergeyt** [comment]
@raphox please feel free to make PR for `async` option.

**Comment by raphox** [comment]
The data-async attribute is not considered. See more about in https://github.com/sergeyt/meteor-typeahead/blob/master/index.js#L75

You may want to consider async when the `data-source` attribute is string.

I will try implement a new PR.

Thank you @sergeyt .

**Pull request merged**